### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by SoorajModi**
Describe what the changes are meant to address. If the implementation is non trivial, please also include a short overview.

Our PR template was really wordy, I made it more concise and easier to fill our by devs and easier to understand by reviewers

---

Fixes: n/a
Closes: n/a
Related: n/a

---

Checklist:

- [ ] Add automated tests cases
- [X] Run `npm run lint-fix` and ensure linter is still passing
- [X] Verify that all automated tests pass
- [ ] Verify that the changes work as expected on Discord
- [X] Update the README and other applicable documentation pages
- [ ] Update the changelog

### CODE REVIEW

**Comment by SoorajModi** [comment]
This one is also not behind by 100 commits XD

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,24 +1,39 @@
...
+
+[insert text here]
 
 ---
 
 <!-- Tags (fill and keep as many as applicable): -->
 
 Fixes: #number of issue (implies Closes tag) or commit SHA
+
 Closes: #number of issue or pull request
```
Comment (MarshallAsch) [inline .github/pull_request_template.md:25]: Can this be moved to being first, I find it odd that its after the description. 

IMO the issue it closes I would expect to be first.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,24 +1,39 @@
-Describe what the changes are meant to address. If the implementation is non trivial, please also include a short overview.
+**Select one or more of:**
 
+<!-- To check an item, fill the brackets with the letter x; the result should look like `[x]`. -->
 
+- [ ] Issue
+- [ ] Feature
+- [ ] Fix
+- [ ] Housekeeping
+- [ ] Other:
```
Comment (MarshallAsch) [inline .github/pull_request_template.md:9]: Can this be done with a label instead?

**Review by MarshallAsch — CHANGES_REQUESTED** [review]
A few comments but once those are addressed it seems good, it also seems wordier now than it was before

**Comment by SoorajModi** [comment]
I thought about it some more and I think this is much improved

File: .github/pull_request_template.md (hunk 2 above)
Comment (SoorajModi) [reply .github/pull_request_template.md]: How would that look? Asking cuz im actually facing this problem at work

File: .github/pull_request_template.md (hunk 1 above)
Comment (SoorajModi) [reply .github/pull_request_template.md]: Yup, I agree, moved it to first in my latest push

**Review by MarshallAsch — APPROVED** [review]

**Pull request merged**